infrastructure/demo.tfvars

```diff
@@ -0,0 +1,3 @@
+capacity = "1"
+instance_size="I1"
+enable_ase = true
```

Comment:
remove the `enable_ase` flag.  We def don't need that.

Reply:
removed